### PR TITLE
Models by moaaz

### PR DESCRIPTION
**Pull request by MoaazMoustafa**
Helloooooooooooooooooooooo

### CODE REVIEW

**Comment by HassanTAli** [comment]
received

**Comment by kataya1** [comment]
Closes #14

**Review by Kholoud731 — APPROVED** [review]
Reviewed

File: backend/backend/accounts/models.py [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
-
-
-#model history
-
+from django.contrib.auth.models import AbstractUser
+from pins.models import Topic
+
+
+class User(AbstractUser):
+    dateOfBirth = models.DateField(null=True)
```
Comment (kataya1) [inline backend/backend/accounts/models.py:7]:
```suggestion
    date_of_birth = models.DateField(null=True)
```

File: backend/backend/accounts/models.py [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+    gender = models.CharField(
+        choices=(('male', 'm'), ('female', 'f')), max_length=50, default="male")
+    # d will be lowercase
+    avatar = models.ImageField(
+        upload_to="profilePic/%y/%m/%d", null=True, blank=True)
+    bio = models.TextField(null=True, blank=True)
+    # add interset as many to many field
+    interest = models.ManyToManyField(Topic)
+    # first issue
+    followers = models.ManyToManyField(
```
Comment (kataya1) [inline backend/backend/accounts/models.py:21]: following = models.ManyToManyField("self", related_name="followers", null=True, blank=True)

**Review by SaharMamdouh — APPROVED** [review]

File: backend/backend/accounts/models.py (hunk 2 above)
Comment (kataya1) [inline backend/backend/accounts/models.py:21]:
```suggestion
      following = models.ManyToManyField("self", related_name="followers", null=True, blank=True)
 
```

File: backend/backend/accounts/models.py [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+        'self', related_name='follower', blank=True)
```
Comment (kataya1) [inline backend/backend/accounts/models.py:22]:
```suggestion
```

**Pull request merged**